modules/nw-operator-cr.adoc

```diff
@@ -289,6 +289,20 @@ endif::ibm-cloud[]
 ====
  While migrating egress traffic, you can expect some disruption to workloads and service traffic until the Cluster Network Operator (CNO) successfully rolls out the changes.
 ====
+
+|`v4InternalSubnet`
+|
+If your existing network infrastructure overlaps with the `100.64.0.0/16` IPv4 subnet, you can specify a different IP address range for internal use by OVN-Kubernetes. You must ensure that the IP address range does not overlap with any other subnet used by your {product-title} installation. The IP address range must be large enough to allocate one IP address per cluster node.
```

Comment:
To be precise, the IP address range must be larger than the maximum number of nodes that can be added to the cluster.

For example, if the clusterNetwork CIDR is 10.128.0.0/14, and the hostPrefix is /23. The maximum number of nodes is 2^(23-14)=128. And as we need one IP for the gateway and one for broadcasting. So the internal IP range must be at least `/24`.

Reply:
There is an error in the calculation. The maximum number of nodes is 2^(23-14)=512, So the internal IP range must be at least /22.